IssueSyncTool/sync_issue.py

```diff
@@ -318,11 +318,16 @@ def process_new_issue(issue, des_tracker, assignee, component_mapping=None):
    if component_mapping:
       if issue.component and issue.component in component_mapping:
          des_title = f"[ {component_mapping[issue.component]} ] {issue.title}"
+   
```

Comment:
Hi Ngoan,
hm... I let the Assignee open, as lang as the ticket is in the backlog.
Actually I don't want an assignee in RTC, too.
Is this causing an issue?
Thank you,
 Thomas

Reply:
Hi Thomas,

It is fine to let assignee open (unassigned).
I have updated rtc implementation to let it unassigned instead of the logged in user (who execute the tool) as assignee if it is missing in the original ticket.

Thank you,
Ngoan